<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Employee">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Full Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phonenumber</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dataset</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="dataSet_IMG">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="btn_Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Department">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>450</width>
      <height>500</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Full Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phonenumber</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="txt_Search">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>111</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search here</string>
    </property>
   </widget>
   <widget class="QLabel" name="btn_Search">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
